components/Section.tsx

```diff
@@ -7,7 +7,7 @@ interface PropsInterface {
 export default function Section(props: PropsInterface) {
     const { children, } = props;
```

Comment:
`children,`

remove commas, do we hava prettierrc already? if no lets create one for uniformity